xds/internal/xdslbregistry/converter_test.go

```diff
@@ -366,7 +362,7 @@ func (s) TestConvertToServiceConfigFailure(t *testing.T) {
 
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
-			_, gotErr := xdslbregistry.ConvertToServiceConfig(test.policy)
+			_, gotErr := ConvertToServiceConfig(test.policy)
```

Comment:
Did we wanna make this test be `package xdslbregistry_test` and keep this?  Seems worth it.

Reply:
Why? To test at the API layer? The main reason we had this in separate package initially was to not import LB Configs.

Reply:
> To test at the API layer?

Yes?  It seems we don't need any internals, so it seems better to not be in the same package unless necessary.

Reply:
K switched.

Reply:
This can go in the same directory.  Please move it back.

Reply:
I deleted this file and moved to e2e test.